lib/platform/gitlab/index.ts

```diff
@@ -161,7 +161,7 @@ export async function getRawFile(
   branchOrTag?: string
 ): Promise<string | null> {
   const escapedFileName = urlEscape(fileName);
-  const repo = repoName ?? config.repository;
+  const repo = urlEscape(repoName || config.repository);
```

Comment:
```suggestion
  const repo = urlEscape(repoName ?? config.repository);
```

Reply:
I'm curious though, what "falsely" value would you want to preserve here?

Reply:
Thanks for looking at this PR!  Personally, I wouldn't use nullish coalescing here, as there are no falsey values that I think the app would benefit from.  It would cause more errors as empty strings, NaN, 0, and false would succeed if they're on the left hand side.   As such, the logical OR operator makes more sense here.

Reply:
We'll get lint issues with `||` when enabling strict null check soon.

I don't want to preserve wrong values, these should simply never happen. Otherwise we should accept them on function arg type. 😉

Reply:
> I don't want to preserve wrong values, these should simply never happen. Otherwise we should accept them on function arg type. 😉

Hah, you got me there on the static typing.

> We'll get lint issues with || when enabling strict null check soon.

Given this, I'm fine with letting you two decide whether to go with nullish coalescing or not. Since you two are maintainers, you'll have a better picture of what fit better with the repo's long term vision. 👍